MIGRATION.md

```diff
@@ -2160,6 +2160,7 @@ addParameters({
       base: 'light',
       brandTitle: 'Storybook',
       brandUrl: 'https://storybook.js.org',
+      brandTarget: '_self',
```

Comment:
I appreciate the attempt at comprehensiveness, but this section of the Migration Guide is about version `5.0`, whereas the code changes in this PR would be released in `6.4.21` & the latest `6.5 alpha`. As such, I don't think it should be documented here, but rather in the actual documentation, here: https://github.com/storybookjs/storybook/blob/next/docs/snippets/common/storybook-theme-example-variables.ts.mdx, which is embedded in [this docs page](https://github.com/storybookjs/storybook/blob/next/docs/configure/theming.md).

Reply:
Removed unnecessary fix in MIGRATION.md